### PR TITLE
UI Setting Tile

### PR DESCRIPTION
**Pull request by AbdullahAhmedSoomro**
- Setting Tile Completed

Isme Color direct bhi use howe hain uski wajah yeh k humne aik theme extension banayi hai issue #298 , to yeh jb complete hogi to is me changing hogi

### CODE REVIEW

File: lib/presentation/shared/tile.dart [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    Key? key,
+    required this.title,
+    required this.subtitle,
+    required this.onTap,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: MediaQuery.of(context).size.width * 0.95,
```
Comment (hashirshoaeb) [inline lib/presentation/shared/tile.dart:18]: why this sizeBox?

**Comment by hashirshoaeb** [comment]
pushed a commit.

**Comment by hashirshoaeb** [comment]
post ui screenshot

**Review by hashirshoaeb — APPROVED** [review]

File: lib/presentation/shared/tile.dart (hunk 1 above)
Comment (hashirshoaeb) [reply lib/presentation/shared/tile.dart:18]: keep the tile expanded, and just use horizontal padding.

Comment (AbdullahAhmedSoomro) [reply lib/presentation/shared/tile.dart:18]: Okay
Remove splash and highlight color or not?

https://user-images.githubusercontent.com/50518325/218433404-30f22b6e-08f6-49d2-8211-72f2c054b7ba.mp4

Comment (hashirshoaeb) [reply lib/presentation/shared/tile.dart]: keep

**Review by hashirshoaeb — APPROVED** [review]

**Comment by hashirshoaeb** [comment]
you can merge it.

**Pull request merged**